Gems/PhysX/Code/Editor/KinematicDescriptionDialog.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "KinematicDescriptionDialog.h"
+
+AZ_PUSH_DISABLE_DLL_EXPORT_MEMBER_WARNING
+#include <Editor/ui_KinematicDescriptionDialog.h>
+AZ_POP_DISABLE_DLL_EXPORT_MEMBER_WARNING
+
+static constexpr AZStd::string_view dynamicDescription = "With <b>Dynamic</b> rigid bodies, you can physics forces "
+                                                  "(gravity, collision, etc.) to control the movement and "
+                                                  "position of the object.";
+
+static constexpr AZStd::string_view kinematicDescription = "With <b>Kinematic</b> rigid bodies, you can use Transform to "
+                                                    "control the movement and position of the object. ";
+
+static constexpr AZStd::string_view moveWithCode = "Move with code";
+static constexpr AZStd::string_view moveManually = "Move manually";
+static constexpr AZStd::string_view collisions = "Collisions";
+static constexpr AZStd::string_view gravity = "Gravity";
+
+static constexpr AZStd::string_view impactIcon = "<img src=\":/stylesheet/img/16x16/impact.svg\"/>";
+static constexpr AZStd::string_view gravityIcon = "<img src=\":/stylesheet/img/16x16/gravity.svg\"/>";
+static constexpr AZStd::string_view moveManuallyIcon = "<img src=\":/stylesheet/img/16x16/move_manually.svg\"/>";
+static constexpr AZStd::string_view moveWithCodeIcon = "<img src=\":/stylesheet/img/16x16/move_with_code.svg\"/>";
+
+KinematicDescriptionDialog::KinematicDescriptionDialog(bool kinematicSetting, QWidget* parent)
+    : QDialog(parent)
+    , m_ui(new Ui::KinematicDescriptionDialog)
+    , m_kinematicSetting(kinematicSetting)
+{
+    m_ui->setupUi(this);
+
+    InitializeButtons();
+    UpdateDialogText();
+}
+
+KinematicDescriptionDialog::~KinematicDescriptionDialog()
+{
+}
+
+void KinematicDescriptionDialog::InitializeButtons()
+{
+    connect(m_ui->dynamicRadioButton, &QPushButton::clicked, this, &KinematicDescriptionDialog::OnButtonClicked);
+    connect(m_ui->kinematicRadioButton, &QPushButton::clicked, this, &KinematicDescriptionDialog::OnButtonClicked);
+    connect(m_ui->dynamicBox, &QGroupBox::clicked, this, &KinematicDescriptionDialog::OnButtonClicked);
+    connect(m_ui->kinematicBox, &QGroupBox::clicked, this, &KinematicDescriptionDialog::OnButtonClicked);
+
+    m_kinematicSetting ? m_ui->kinematicRadioButton->setChecked(true) : m_ui->dynamicRadioButton->setChecked(true);
```

Comment:
I find it odd to use ternary operator for branching without being assigned to something, so it reads like something it's missing and leaves it wondering if it's intentional of if `setChecked` actually returns something but it's just an error not assign it to a variable. For these cases I would just use a regular `if` block.

````
if (m_kinematicSetting )
{
    m_ui->kinematicRadioButton->setChecked(true);
}
else
{
    m_ui->dynamicRadioButton->setChecked(true);
}
````